api/v1alpha1/driver_types.go

```diff
@@ -166,6 +172,9 @@ type NodePluginSpec struct {
 	// Control the host mount of /etc/selinux for csi plugin pods. Defaults to false
 	//+kubebuilder:validation:Optional
 	EnableSeLinuxHostMount *bool `json:"EnableSeLinuxHostMount,omitempty"`
+	// Topology settings for the plugin pods
+	//+kubebuilder:validation:Optional
+	Topology *TopologySpec `json:"topology,omitempty"`
```

Comment:
Is topology only important for NodePlugin? what about the ControllerPlugin? shouldn't er have that in the PodCommonSpec?

Reply:
This is only required for node plugin they need to advertise topology on the node where it is running

Reply:
ack